src/main/kotlin/no/nav/syfo/sykmeldingstatus/kafka/producer/SykmeldingStatusKafkaProducer.kt

```diff
@@ -1,18 +1,18 @@
 package no.nav.syfo.sykmeldingstatus.kafka.producer
 
-import java.time.LocalDateTime
+import java.time.ZonedDateTime
 import no.nav.syfo.log
-import no.nav.syfo.sykmeldingstatus.kafka.model.KafkaMetadata
-import no.nav.syfo.sykmeldingstatus.kafka.model.SykmeldingStatusKafkaEvent
-import no.nav.syfo.sykmeldingstatus.kafka.model.SykmeldingStatusKafkaMessage
+import no.nav.syfo.model.sykmeldingstatus.KafkaMetadataDTO
+import no.nav.syfo.model.sykmeldingstatus.SykmeldingStatusKafkaEventDTO
+import no.nav.syfo.model.sykmeldingstatus.SykmeldingStatusKafkaMessageDTO
 import org.apache.kafka.clients.producer.KafkaProducer
 import org.apache.kafka.clients.producer.ProducerRecord
 
-class SykmeldingStatusKafkaProducer(private val kafkaProducer: KafkaProducer<String, SykmeldingStatusKafkaMessage>, private val topicName: String) {
-    fun send(sykmeldingStatusKafkaEvent: SykmeldingStatusKafkaEvent) {
-        log.info("Skriver statusendring for sykmelding med id {} til topic", sykmeldingStatusKafkaEvent.sykmeldingId)
-        val metadata = KafkaMetadata(sykmeldingStatusKafkaEvent.sykmeldingId, LocalDateTime.now())
-        val sykmeldingStatusKafkaMessage = SykmeldingStatusKafkaMessage(metadata, sykmeldingStatusKafkaEvent)
-        kafkaProducer.send(ProducerRecord(topicName, sykmeldingStatusKafkaMessage.event.sykmeldingId, sykmeldingStatusKafkaMessage))
+class SykmeldingStatusKafkaProducer(private val kafkaProducer: KafkaProducer<String, SykmeldingStatusKafkaMessageDTO>, private val topicName: String) {
+    fun send(sykmeldingStatusKafkaEventDTO: SykmeldingStatusKafkaEventDTO, source: String) {
+        log.info("Skriver statusendring for sykmelding med id {} til topic", sykmeldingStatusKafkaEventDTO.sykmeldingId)
+        val metadataDTO = KafkaMetadataDTO(sykmeldingId = sykmeldingStatusKafkaEventDTO.sykmeldingId, timestamp = ZonedDateTime.now(), source = source)
```

Comment:
her vil det også bli skrevet til topic på dette formatet "+01:00[Europe/Oslo]"
+"01:00" er forsåvidt innafor standarden men ikke "[Europe/Oslo]" så enten må serializern konfigureres slik at den dropper det siste eller så er det lettere å bare bruke .now(ZoneOffset.UTC)

Reply:
Kanskje vi burdeBruke OffsetDateTime så slipper vi feks "[Europe/Oslo]"

Reply:
I stedet for ZonedDateTime? Da må vi endre overalt en gang til, både her, i kafkaformatet og i registeret. Vi kan gjøre det hvis det er riktig, men da må vi være HELT sikre denne gangen.

Reply:
Ja det er egentlig ikke så viktig om vi bruker ZonedDateTime eller OffsetDateTime så lenge vi skriver til kafka (eller tilbyr api) på ISO 8601.

Reply:
Hvordan får jeg ISO 8601? Har ikke ISO-standardene i hodet :P